### PR TITLE
Feature/send quiz view

### PR DESCRIPTION
**Pull request by Derstilon**
I've added a view that enables sending quizzes to selected students for a selected amount of time and with a clear display of what options you have chosen.
image
image
image
image

### CODE REVIEW

File: src/lecturer/components/studentListView/StudentListView.tsx [hunk 1]
```diff
@@ -19,75 +31,102 @@ export function StudentListView(props: StudentListViewProps) {
...
+        event: React.MouseEvent<unknown>,
+        property: keyof StudentListRow
+    ) => {
+        const isAsc = orderBy === property && order === "asc";
+        setOrder(isAsc ? "desc" : "asc");
         setOrderBy(property);
     };
 
+    const handleToggle = (value: string) => () => {
+        console.log(studentList);
```
Comment (michal367) [inline src/lecturer/components/studentListView/StudentListView.tsx:105]: remove all unecessery console.logs

**Review by michal367 — COMMENTED** [review]
You can make it better. This x and checkbox don't look good. Maybe checkbox to new line?
obraz

File: src/lecturer/components/studentListView/StudentListView.tsx (hunk 1 above)
Comment (Derstilon) [reply src/lecturer/components/studentListView/StudentListView.tsx]: Done

**Review by Petros9 — APPROVED** [review]

**Comment by Derstilon** [comment]
> You can make it better. This x and checkbox don't look good. Maybe checkbox to new line?
> obraz

image

**Review by michal367 — APPROVED** [review]
Really great job!

**Pull request merged**